### PR TITLE
New VM flavor, with custom opcodes, for EIP150

### PR DESCRIPTION
**Pull request by gsalgado**

### CODE REVIEW

**Comment by pipermerriam** [comment]
Ready for review?

File: evm/logic/system.py [hunk 1]
```diff
@@ -52,7 +74,7 @@ def create(computation):
 
     call_data = computation.memory.read(start_position, size)
 
-    create_msg_gas = computation.gas_meter.gas_remaining
+    create_msg_gas = max_child_gas_modifier(computation.gas_meter.gas_remaining)
```
Comment (pipermerriam) [inline evm/logic/system.py]: This doesn't sit well with me.  Functions which pass around methods strike me as harder to follow than using a class.  Do you think there is a cleaner way to do this with a class abstraction similar to how it's done with the `Call` classes?

File: evm/vm/flavors/eip150/opcodes.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    context,
+    storage,
+    system,
+)
+
+from evm.vm.flavors.homestead.opcodes import HOMESTEAD_OPCODES
+
+
+NEW_OPCODES = {
+}
```
Comment (pipermerriam) [inline evm/vm/flavors/eip150/opcodes.py]: Can we drop `NEW_OPCODES` since it's empty?

Comment (gsalgado) [inline evm/vm/flavors/eip150/opcodes.py]: Sure, removed

File: evm/logic/system.py (hunk 1 above)
Comment (gsalgado) [inline evm/logic/system.py]: The last commit changes it to use a class like Call. I don't find it significantly cleaner, but I don't feel strong about either way

**Pull request merged**